dev/Repeater/ItemsRepeater.cpp

```diff
@@ -118,6 +118,18 @@ winrt::Size ItemsRepeater::MeasureOverride(winrt::Size const& availableSize)
         virtualContext->Indent(Indent());
 #endif
 
+        // Checking if we have an data template
+        if (m_dataTemplateReference) {
+            // Checking if it has no content
+            if (!m_dataTemplateReference.LoadContent().as<winrt::FrameworkElement>()) {
+                // Has no content, so we will draw nothing and request zero space
+                extent = winrt::Rect{ m_layoutOrigin.X, m_layoutOrigin.Y, 0, 0 };
+                m_viewportManager->SetLayoutExtent(extent);
+                m_lastAvailableSize = availableSize;
+                return desiredSize;
```

Comment:
You will still need to run the logic below to recycle candidate elements.

Reply:
I have removed the return statement and removed the lines that were duplicate.